Goobi/src/de/sub/goobi/beans/Prozess.java

```diff
@@ -466,7 +466,7 @@ public Boolean getTifDirectoryExists() {
 		if (testMe.list() == null) {
 			return false;
 		}
-		if (testMe.exists() && testMe.list().length > 0) {
```

Comment:
And then…? Add a `catch (NullPointerException`? Not the nearest solution either, is it?